client/components/videos-ui/video-modal.jsx

```diff
@@ -19,10 +16,11 @@ const BloggingQuickStartModal = ( props ) => {
 
 	return (
 		isVisible && (
-			<BlankCanvas className={ 'blogging-quick-start-modal' }>
+			// eslint-disable-next-line wpcalypso/jsx-classname-namespace
+			<BlankCanvas className="video-modal">
```

Comment:
Let's stick with the [CSS guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md):

```suggestion
			<BlankCanvas className="videos-ui__modal">
```

You can also remove the `esling-disable-next-line`, and don't forget to update the `styles.scss` file! 🙂